.github/ISSUE_TEMPLATE/release-tracking-issue.md

```diff
@@ -36,9 +36,14 @@ Note: remove the backticks (``) from the link.
 - [ ] Release <RELEASE_NUMBER> pushed to production (<CL submit time>)
 
 <!--
+If this release will become the next LTS release, add/update the checkboxes above as needed e.g.
+
+- [x] Release <RELEASE_NUMBER> will be the next LTS
+- [ ] Release <RELEASE_NUMBER> pushed to LTS (<CL submit time>)
```

Comment:
Having both lines is redundant, no?

Reply:
I don't believe so. The first would be checked when a build is first release as experimental/beta (or unchecked). Maybe the first one should always be present. The second one is only relevant if the first one is checked, but that's for indicating that the release has been promoted to LTS. So there would be a 2 week period where the first is checked but not the second.

Reply:
My 2 cents: I think the presence of line 42 is enough to indicate that it will be the next LTS, and makes line 41 unnecessary.

Reply:
Will address in #26256